lib/iris/util.py

```diff
@@ -1080,6 +1080,10 @@ def new_axis(src_cube, scalar_coord=None):
     if scalar_coord is not None:
         scalar_coord = src_cube.coord(scalar_coord)
 
+    if src_cube.shape is ():
```

Comment:
@ajdawson This all makes sense, I am happy to implement these changes.  

I am not sure if I've tried this with lazy data.  I assume if I write a test using a lazy array this would also just be a unit test?